opal/mca/pmix/pmix1xx/pmix/examples/server.c

```diff
@@ -343,6 +346,20 @@ static void errhandler(pmix_status_t status,
     pmix_output(0, "SERVER: ERRHANDLER CALLED WITH STATUS %d", status);
 }
 
+static void op_callbk(pmix_status_t status,
+                      void *cbdata)
+{
+    pmix_output(0, "SERVER: OP CALLBACK CALLED WITH STATUS %d", status);
+}
+
+static void errhandler_reg_callbk (pmix_status_t status,
+                                   int errhandler_ref,
+                                   void *cbdata)
+{
+    pmix_output(0, "SERVER: ERRHANDLER REGISTRATION CALLBACK CALLED WITH STATUS %d, ref=%d",
```

Comment:
We should probably remove this debug, or at least replace it with pmix_output_verbose